src/Cards/CardManager.ts

```diff
@@ -178,19 +178,26 @@ export class CardManager {
 	/**
 	 * Fetches collection of cards with the given `identity` from the Virgil
 	 * Cards Service.
-	 * @param {string} identity - Identity of the cards to fetch.
+	 * @param {string|string[]} identities - Identity or an array of identities of the cards to fetch.
 	 * @returns {Promise<ICard[]>}
 	 */
-	async searchCards (identity: string): Promise<ICard[]> {
-		const tokenContext: ITokenContext = { operation: 'search' };
+	async searchCards (identities: string|string[]): Promise<ICard[]> {
+		if (!identities) throw new Error('Argument `identities` is required');
```

Comment:
It's better to throw `TypeError` here
```suggestion
		if (!identities) throw new TypeError('Argument `identities` is required');
```